src/test/java/com/kodilla/ecommercee/ProductTests.java

```diff
@@ -0,0 +1,136 @@
+package com.kodilla.ecommercee;
+
+import com.kodilla.ecommercee.domain.Product;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.transaction.annotation.Transactional;
+
+
+@Transactional
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class ProductTests {
+
+    @Autowired
+    private GenericEntityRepository genericEntityRepository;
+
+
+    @Test
+    public void testSaveProduct() {
+
+        //Given
+        Product product = new Product("Altana SunSet", "Altana ogrodowa ...", 999.90);
```

Comment:
Wydaje mi się,że to co w tej chwili jest w linii 29 i 32 powinno być jeszcze w sekcji given,to co w linii 33 w sekcji when

Reply:
@aleksandra4991 Racja, poprawię.